FreeRTOS/FreeRTOS-Plus-TCP
# Free old network buffers once its duplicated

Description
-----------
This PR fixes the network buffers being lost when they are duplicated inside the `vNDSendNeighbourSolicitation` and `vNDSendRouterSolicitation` by releasing the old buffers.

Test Steps
-----------
Tested ND 

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
